custom-elements.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 2,
+  "tags": [
+    {
+      "name": "modal-manager",
+      "description": "A component with a title and an action counter",
+      "properties": [
+        {
+          "name": "title",
+          "type": "String",
+          "description": "The title of your component",
+          "default": "Hey there"
+        },
+        {
+          "name": "counter",
+          "type": "Number",
+          "description": "An action counter",
+          "default": 0
+        }
+      ],
+      "events": [],
+      "slots": [],
+      "cssProperties": [
+        {
+          "name": "--modal-manager-text-color",
```

Comment:
I don't see this custom variable used in the component code. Looks like there are also a lot of CSS variables that could be defined here and/or in the readme.

Reply:
I updated this with the element details.